docs/apis-tools/camunda-api-rest/tutorial.md

```diff
@@ -0,0 +1,271 @@
+---
+id: camunda-8-api-tutorial
+title: Tutorial
+description: "New to the Camunda 8 API? Step through our tutorial to list all roles, create a role, retrieve a role, and delete a role."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Camunda 8 API, such as listing all roles, creating a role, retrieving a role, and deleting a role.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/guides/create-cluster.md).
+- Upon cluster creation, [create your first client](/guides/setup-client-connection-credentials.md). Ensure you check the `Zeebe` client scope box.
+
+:::note
+Make sure you keep the generated client credentials in a safe place. The **Client secret** will not be shown again. For your convenience, you can also download the client information to your computer.
+:::
+
+- In this tutorial, we utilize a JavaScript-written [GitHub repository](https://github.com/camunda/camunda-api-tutorials) to write and run requests. Clone this repo before getting started.
+- Ensure you have [Node.js](https://nodejs.org/en/download) installed as this will be used for methods that can be called by the CLI (outlined later in this guide). Run `npm install` to ensure you have updated dependencies.
+
+## Getting started
+
+- You need authentication to access the API endpoints. Find more information [here](./camunda-api-rest-authentication.md).
+
+## Set up authentication
+
+If you're interested in how we use a library to handle auth for our code, or to get started, examine the `auth.js` file in the GitHub repository. This file contains a function named `getAccessToken` which executes an OAuth 2.0 protocol to retrieve authentication credentials based on your client ID and client secret. Then, we return the actual token that can be passed as an authorization header in each request.
+
+To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. You will need to add your `CAMUNDA_CLIENT_ID`, `CAMUNDA_CLIENT_SECRET`, `CAMUNDA_REST_ADDRESS` (after creating a client and downloading the .env variables, this is reflected in the Console UI as `ZEEBE_REST_ADDRESS`), and `CAMUNDA_TOKEN_AUDIENCE` (represented as `ZEEBE_TOKEN_AUDIENCE` in the Console UI), which is `zeebe.camunda.io` in a Camunda 8 SaaS environment. For example, your audience may be defined as `CAMUNDA_TOKEN_AUDIENCE=zeebe.camunda.io`.
```

Comment:
This updated sentence, "You will need...," does clarify things, but it seems very long. Should it be multiple sentences?

Reply:
I ended up restructuring this paragraph into an unordered list. WDYT?